www/img-status-lang.php

```diff
@@ -16,16 +15,12 @@
 $langs = array_keys($LANGUAGES);
 foreach ($langs as $lang) {
     if (!in_array($lang, $available_langs)) {
-        echo "Documentation for $lang language does not exist.\n";
+        die("Information for $lang language does not exist.");
     } else {
         generate_image($lang, $idx);
-        echo "Generated images/revcheck/info_revcheck_php_$lang.png\n";
     }
 }
```

Comment:
I don't think this is right, as all images are now generated for the first language in the available lang array (German). I propose this patch: https://github.com/php/web-doc/pull/55